### ISSUE TITLE
CLI option for localization

### ISSUE THREAD

**Issue by Djcvhw** [john-doherty/selenium-cucumber-js#63]
Hello. How can I change global language for features?
In `cucumber-js` I can use cli options `--language`  but in `selenium-cucumber-js` I use `# language: ru` at first line of feature for every feature

**Comment by DMShamonov** [comment]
Yup. And how we can throw `cucumber-js` parameters across `selenium-cucumber-js` in general? 
For example I want use `babel-core` library. In `cucumber-js` I use `--require-module babel-core/register` parameter. How I can do it in `selenium-cucumber-js`?
